### PR TITLE
updated network metrics

### PR DESCRIPTION
**Pull request by npashilkar**
created a new branch cuz i couldn't figure out some git issues like i actually was screaming. changes are to the readme files and network graph. the network metrics function is added to alans construct network graph.

### CODE REVIEW

File: utils/network.py [hunk 1]
```diff
@@ -204,6 +206,80 @@ def plot_network_graph(G: nx.MultiDiGraph):
...
+        )
+
+        file.write(f"density': {density}\n\n")
+
+        file.write(f"communities where k = 5': {communities}\n\n")
+
+
 def construct_network_graph(
     start_year: int, end_year: int, dfs: list[pd.DataFrame]
 ):
```
Comment (averyschoen) [inline utils/network.py:285]: add to the docstring that calling this function triggers the generation of the network metrics txt file

**Review by averyschoen — CHANGES_REQUESTED** [review]
quick documentation fix

File: utils/network.py [hunk 2]
```diff
@@ -214,6 +290,8 @@ def construct_network_graph(
         dfs: dataframes in the order: inds_df, orgs_df, transactions_df
 
     Returns:
+        creates a network metrics file, plots a network graph,
```
Comment (averyschoen) [inline utils/network.py]: this is misleading -- the function doesn't return anything. Use the body of the docstring to describe what the function does.

File: utils/network.py [hunk 3]
```diff
@@ -225,5 +303,6 @@ def construct_network_graph(
         [inds_df, orgs_df, transactions_df]
     )
     G = create_network_graph(aggreg_df)
+    network_metrics(G)
     plot_network_graph(G)
     nx.write_adjlist(G, "Network Graph Node Data")
```
Comment (averyschoen) [inline utils/network.py]: This seems unnecessary to write out

**Review by averyschoen — CHANGES_REQUESTED** [review]

File: utils/network.py (hunk 3 above)
Comment (npashilkar) [reply utils/network.py]: i need to call the function using the graph object, does it need to be written elsewhere?

Comment (averyschoen) [reply utils/network.py]: this was specifically on the nx.write)adjlist(G, "Network Graph Node Data") line

Comment (naynapashilkar) [reply utils/network.py]: oh, that i'm not sure about that then it's alan's code. i can ask him

**Review by averyschoen — APPROVED** [review]

**Pull request merged**